starter_code/src/movies.js

```diff
@@ -1,13 +1,44 @@
+"use strict";
 /* eslint no-restricted-globals: 'off' */
 // Turn duration of the movies from hours to minutes 
 
 
 // Get the average of all rates with 2 decimals 
 
+//var decimalNumber = parseFloat(movies[rate].toFixed(2));
+
+function ratesAverage(array){
+    var total = array.reduce(function (accumulator, item) {
+        return accumulator + Number(item.rate); 
+    },0).toFixed(2);
+    return total/array.length;
+}
+ratesAverage(movies);
 
 // Get the average of Drama Movies
 
 
+var total = array.reduce(function(accumulator, item) {
```

Comment:
'array' is not defined, it doesn't exist in your scope. However, if you put this code inside the function dramaMoviesRate, then it will work, because array is its parameter. Please, ask me if this is not clear.